resources/Makefile.j2

```diff
@@ -0,0 +1,111 @@
+	test -d {{ config.configuration }}/hp-mp-semsim.{{ config.configuration }} || cp -rf configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/hp-mp-semsim.{{ config.configuration }}
+	java -cp $(H2_JAR) org.h2.tools.RunScript -user sa -url jdbc:h2:/configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/hp-mp-semsim.{{ config.configuration }}/{{ phenotype.version }}_phenotype -script configurations/{{ config.configuration }}/hp-mp.semsim.{{ config.configuration }}.sql
+
+.PHONY: semsim-ingest
+semsim-ingest: configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/{{ phenotype.version }}_phenotype.h2.db
+
+{% endif %}
+
+{% endfor %}
+
+.PHONY: prepare-corpus
+
+{% for run in runs %}
+results/{{ run.tool }}-{{ run.version }}/{{ run.configuration }}-{{ run.corpus }}-{{ run.corpusvariant }}/results.yml: configurations/{{ run.tool }}-{{ run.version }}-{{ run.configuration }}/config.yaml
+	
+	pheval run \
+	 --input-dir {{ directories.config }}/{{ run.tool }}-{{ run.version }}-{{ run.configuration }} \
+	 --testdata-dir $(shell pwd)/corpora/{{ run.tool }}/{{ run.corpus }}/{{ run.corpusvariant }} \
+	 --runner {{ run.tool }}phevalrunner \
+	 --tmp-dir {{ directories.tmp }}/ \
+	 --output-dir $(shell dirname pwd)/$(shell dirname $@) \
+	 --config $<
+	
+	touch $@
+
+
+corpora/{{ run.tool }}/{{ run.corpus }}/{{ run.corpusvariant }}/corpus.yml: $(TEST_DATA)/template_vcf/template_exome_hg19.vcf.gz
+	{% if run.tool == 'exomiser' %}
+	test -d $(shell dirname $@)/vcf || mkdir -p $(shell dirname $@)/vcf
+	test -L $(shell pwd)/corpora/{{ run.tool }}/{{ run.corpus }}/{{ run.corpusvariant }}/template_exome_hg19.vcf.gz || ln -s $(shell pwd)/$< $(shell dirname $@)/vcf/
+	pheval-utils create-spiked-vcfs \
+	 --template-vcf-path $(shell pwd)/$(shell dirname $@)/vcf/template_exome_hg19.vcf.gz \
+	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/single \
+	 --output-dir $(shell pwd)/$(shell dirname $@)/vcf
+	{% endif %}
+
+	test -d $(shell dirname $@)/phenopackets || mkdir -p $(shell dirname $@)/phenopackets
+	pheval-utils scramble-phenopackets \
+	 --scramble-factor 1 \
+	 --output-dir $(shell pwd)/$(shell dirname $@)/phenopackets \
+	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/single
+	touch $@
+
+
+prepare-corpus +=  corpora/{{ run.tool }}/{{ run.corpus }}/{{ run.corpusvariant }}/corpus.yml
+
+
+
+run-{{ run.corpus }}-{{ run.tool }}:
+	$(MAKE) SCRAMBLE_FACTOR=1 results/{{ run.tool }}-{{ run.version }}/{{ run.configuration }}-{{ run.corpus }}-{{ run.corpusvariant }}/results.yml
```

Comment:
Why is the SCRAMBLE_FACTOR here? It should be in the config

Reply:
And actually, it should not be necessary at all, because it is in `{{ run.corpusvariant }}`